src/Button/SimpleButton/SimpleButton.jsx

```diff
@@ -30,25 +27,48 @@ class SimpleButton extends React.Component {
    * @type {Object}
    */
   static propTypes = {
+    className: PropTypes.string,
     /**
-     * The className which should be added.
+     * The font awesome icon name.
      * @type {String}
      */
-    className: PropTypes.string,
     icon: PropTypes.string,
+    /**
+     * The classname of an icon of an iconFont. Use either this or icon.
+     * @type {String}
+     */
     fontIcon: PropTypes.string,
-    onClick: PropTypes.func,
+    /**
+     * The tooltip to be shown on hover.
+     * @type {String}
+     */
     tooltip: PropTypes.string,
-    tooltipPlacement: PropTypes.string
+    /**
+     * The position of the tooltip.
+     * @type {String}
+     */
+    tooltipPlacement: PropTypes.oneOf([
+      'top',
+      'left',
+      'right',
+      'bottom',
+      'topLeft',
+      'topRight',
+      'bottomLeft',
+      'bottomRight',
+      'leftTop',
+      'leftBottom',
+      'rightTop',
+      'rightBottom'
+    ])
```

Comment:
Is there some default value for placement? Should it be required?

Reply:
Default is top. See https://ant.design/components/tooltip/